File: engine/model/src/main/scala/tools/packet/messenger/MessengerInvite.scala
package tools.packet.messenger

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MessengerInvite(private var _characterNameFrom: String, private var _messengerId: Int) extends PacketInput {
  def characterNameFrom: String = _characterNameFrom

  def messengerId: Int = _messengerId

  override def opcode(): SendOpcode = SendOpcode.MESSENGER
}

File: engine/engine-model/src/main/java/tools/packet/messenger/MessengerInvite.java
package tools.packet.messenger; 
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MessengerInvite(String characterNameFrom, Integer messengerId) implements PacketInput {
  @Override
  public SendOpcode opcode() {
    return SendOpcode.MESSENGER;
  }
}